providers/openshift/provider.go

```diff
@@ -399,6 +413,49 @@ func (p *OpenShiftProvider) reviewUser(name, accessToken string) error {
 	return nil
 }
 
+func (p *OpenShiftProvider) GetUserProjects(s *providers.SessionState) (projs []string, err error) {
```

Comment:
Do not use bare returns or named returned parameters.  They have some value in generated code and defer/panic flows, but otherwise make it significantly harder to reason about what a function will return, especially when the named returns get shadowed.